betterlock/files/betterlock/po/betterlock.pot

```diff
@@ -50,6 +50,9 @@ msgstr ""
 msgid "Show notifications"
 msgstr ""
 
+msgid "Silence system notification sound"
+msgstr ""
+
```

Comment:
There is a script precisely so that these files do not need to be modified manually. If you look in the README, it is explained there. For now, I'm going to run `./cinnamon-spices-makepot betterlock` so that this can be merged.

Reply:
Got it, I'll keep that in mind for the future. Thank you!